test-jest/component/ResponsiveContainer.spec.tsx

```diff
@@ -168,4 +168,26 @@ describe('<ResponsiveContainer />', () => {
     expect(element).toHaveAttribute('width', '100');
     expect(element).toHaveAttribute('height', '100');
   });
+
+  it('should resize when debounced', () => {
+    jest.useFakeTimers('modern');
+    const { container } = render(
+      <ResponsiveContainer id="testing-id-attr" width="100%" height={200} debounce={200}>
+        <div data-testid="inside" />
+      </ResponsiveContainer>,
+    );
+
+    const element = container.querySelector('.recharts-responsive-container');
+
+    notifyResizeObserverChange([{ contentRect: { width: 50, height: 50 } }]);
+    jest.advanceTimersByTime(100);
+    expect(element).not.toHaveAttribute('width');
+    expect(element).not.toHaveAttribute('height');
```

Comment:
Before this change, this check fails because `ReactResizeDetector`'s internal callback is not debounced.